tests/behat/features/page-with-versioned-objects.feature

```diff
@@ -0,0 +1,78 @@
+    And the "Test Page Versioned Object" "My Page" is not published
+    And the "Versioned parent object" "My Versioned Parent Object" with "TestPageVersionedObject"="1"
+    And the "Non Versioned parent object" "My Non Versioned Parent Object" with "TestPageVersionedObject"="1"
+    And the "group" "EDITOR" has permissions "Access to 'Pages' section" and "TEST_DATAOBJECT_EDIT"
+    And I am logged in as a member of "EDITOR" group
+
+  Scenario: I should see versioned badge on page with draft and modified versioned child object
+    Given I go to "/admin/pages"
+      And I should see "My Page" in the tree
+      And I click on "My Page" in the tree
+    Then I should see "My Page" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+      And I click on the "#tab-Root_VersionedParentObjects" element
+      And I should see "My Versioned Parent Object" in the "#Form_EditForm" element
+      And I should see "Draft" in the "#Form_EditForm" element
+    Then I click "My Versioned Parent Object" in the "#Form_EditForm_VersionedParentObjects" element on page
+      And I should see "My Versioned Parent Object" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+      And I click "My Page" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+    Then I click "My Versioned Parent Object" in the "#Form_EditForm_VersionedParentObjects" element on page
+      And I press the "Publish" button
+      And I should not see "Draft" in the ".breadcrumbs-wrapper" element
+      And I click "My Page" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+    Then I click "My Versioned Parent Object" in the "#Form_EditForm_VersionedParentObjects" element on page
+      And I fill in "Name" with "My Versioned Parent Object with changes"
+      And I press the "Save" button
+    Then I should see "Modified" in the ".breadcrumbs-wrapper" element
+      And I click "My Page" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+    Then I press the "Publish" button
+      And I should not see "Modified" in the ".breadcrumbs-wrapper" element
+      And I should not see "Modified" in the "#Form_EditForm" element
+
+  Scenario: I should see versioned badge on page with draft and modified versioned grandchild object
+    Given I go to "/admin/pages"
+      And I should see "My Page" in the tree
+      And I click on "My Page" in the tree
+    Then I should see "My Page" in the ".breadcrumbs-wrapper" element
+      And I should see "Draft" in the ".breadcrumbs-wrapper" element
+      And I click on the "#tab-Root_NonVersionedParentObjects" element
+    Then I click "My Non Versioned Parent Object" in the "#Form_EditForm_NonVersionedParentObjects" element on page
```

Comment:
Update to use the whatever is decided that we use in https://github.com/silverstripe/silverstripe-behat-extension/pull/259#pullrequestreview-1807888699

Also updated any other references in this behat test

Reply:
Updated